server.js

```diff
@@ -4,7 +4,7 @@ const express = require('express');
 const app = express();
 //const bodyParser = require('body-parser)').urlencode({estended: true});
 const PORT = process.env.PORT || 3000;
-app.use(express.static('/portfolio'));
+app.use(express.static('.'));
```

Comment:
Note that this makes *all* files in the current directory available through the web, including `server.js` and `package.json`. In many apps, that sort of exposure is a security vulnerability, which is why we always serve static files out of a specific directory (typically `public/` or `www/`).